lib/elixir/lib/enum.ex

```diff
@@ -1716,7 +1716,7 @@ defmodule Enum do
   @spec slice(t, Range.t) :: list
```

Comment:
Can we define this spec with something like..
`@spec slice(t, integer..integer) :: list`
or it has to be?
`@spec slice(t, ElixirInteger..ElixirInteger) :: list`